crates/uv-requirements/src/pyproject.rs

```diff
@@ -679,13 +695,19 @@ mod test {
         extras: &ExtrasSpecification,
     ) -> anyhow::Result<RequirementsSpecification> {
         let path = uv_fs::absolutize_path(path.as_ref())?;
-        RequirementsSpecification::parse_direct_pyproject_toml(
-            contents,
+        let project_workspace =
+            ProjectWorkspace::dummy(path.as_ref(), &PackageName::from_str("foo").unwrap());
+        let pyproject_toml =
+            toml::from_str(contents).context("Failed to parse: `pyproject.toml`")?;
+        Ok(RequirementsSpecification::parse_direct_pyproject_toml(
+            &pyproject_toml,
+            project_workspace.workspace(),
             extras,
             path.as_ref(),
             PreviewMode::Enabled,
         )
-        .with_context(|| format!("Failed to parse: `{}`", path.user_display()))
+        .with_context(|| format!("Failed to parse: `{}`", path.user_display()))?
+        .unwrap())
```

Comment:
Why unwrap when the method itself already returns `Result`? Can we be consistent here?